docs/user_best_practices.md

```diff
@@ -0,0 +1,251 @@
+
+![Confidence.png](./assets/pics/user-guide/Medium_Confidence_Example.png)
+
+The confidence is a measurement of Scarf's confidence in the IP/metadata -> organization match for each event. Some of our metadata providers like Clearbit provide their own confidence scores and Scarf will take those into account, but we also account for what other providers say. We will make our own adjustments in many cases, for instance, if there is disagreement between the different data providers we use, or if we find irregularities in the metadata.
+
+Confidence intervals have 2 data points associated with them. The first is the overall flag which is low ( red ), medium ( yellow), and green ( high confidence). This gives you a quick way to associate high-probability matches with low ones. When looking into the flags, you will find a % of confidence associated with each company and event, and these percentages can be used to further differentiate which companies and events to prioritize.
+
+| Confidence | Description | How to Use |
+|------------| ----------- | ---------- |
+| Low        | Low confidence matches, based on available data we suspect these events are associated with this company, but can not be 100% sure.  | We do not recommend taking direct action on small numbers of low confidence matches.  Consider these to be very low quality leads, if leads at all. This data however is valuable if correlated with outside data for this organization.  For instance if you know this organization is active in the community or if you see multiple low confidence leads over the course of weeks or months from unique end points, this may indicate these are higher quality then our data suggests.  |
+| Medium     | These events have an above average chance of being from associated companies.  We have been able to match multiple checks in the metadata and our external providers have an above average confidence in their match.  |  Medium confidence matches should be considered second tier data, with leads being followed up after processing higher condfidence data. This data is valuable in enriching other data source and we would feel confident in using it for trends, analysis, and as part of a broader customer intelligence effort. In part of an ABM (account based marketing) strategy we would feel confident in using these matches correlated with a list of targeted accounts. This is also a good list of organizations for BDR and prospecting activities. |             
+| High     |  The events and organizations that have a high confidence interval mean that we have multiple high probability indicators of a match to an organization for these events. |  High confidence matches provide the highest quality data. We are confident in people using this data for their sales pipelines, lead scoring, etc. These records could be integrated into exsiting tools to give a complete picture of a customers journey |
+
+
+<a id="MultiplePackages"></a>

```

Comment:
what is this for?

Reply:
This is code for a linked table of contents, will release in the next iteration